modules/ting_search_fast/ting_search_fast.module

```diff
@@ -0,0 +1,91 @@
+<?php
+
+use Ting\TingObjectCollectionInterface;
+use Ting\TingObjectInterface;
+
+function ting_search_fast_theme($existing, $type, $theme, $path) {
+  return array(
+    'ting_search_fast_item' => array(
+      'variables' => array(
+        'title' => NULL,
+        'creators' => NULL,
+        'abstract' => NULL,
+        'cover' => NULL,
+        'series' => NULL,
+        'availability' => NULL,
+        'rating' => NULL,
+      ),
+      'template' => 'ting-search-fast-item',
+      'path' => $path . '/templates',
+    ),
+  );
+}
+
+/**
+ * Implements hook_theme_registry_alter().
+ */
+function ting_search_fast_theme_registry_alter(&$theme_registry) {
+  $theme_registry['ting_search_collection_snippet']['function'] = 'ting_search_fast_ting_search_collection_snippet';
+}
+
+function ting_search_fast_ting_search_collection_snippet(array $variables) {
+  /* @var \Ting\TingObjectCollectionInterface $collection */
+  $collection = $variables['collection'];
+  $object = $collection->getPrimaryObject();
+
+  // The covers module does not use the search abstraction layer so instead we
+  // create a dummy object with the required properties.
+  $cover_object = new stdClass();
+  $cover_object->id = $object->getId();
+  $cover_object->title = $object->getShortTitle();
+  $cover_object->creators = $object->getCreators();
+
+  return [
+    '#theme' => 'ting_search_fast_item',
+    '#title' => theme('ting_title',
```

Comment:
calling the theme() function directly is strongly discouraged, jfr. api.drupal.org
You could pass a render array here.